### PR TITLE
network policy implementation detail

### PR DESCRIPTION
**Pull request by aojea**
Define implementation details to avoid disruption with current consumers and minimize risks

### CODE REVIEW

File: keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md [hunk 1]
```diff
@@ -497,6 +497,13 @@ An architectural change to the current testing policies has been implemented and
 
 *This implementation runs continuously as part of the Antrea CNI project.*
 
+In order to avoid disruptions and minimize risks and/or regressions, new and old implementation will live in parallel during some time:
+
+- The new network policy framework will be added in a new folder `kubernetes/test/e2e/network/netpol`
+- The new network policy framework will use and additional tag [Feature:Netpol]
+
+Once the new implementation has finished and is stable (it should be able to pass 10/10 jobs consecutively without any code change ), the legacy tests and the additional tag will be removed.
```
Comment (aojea) [inline keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md:505]: by jobs I mean jobs with the same load as we are running in the CI, running all the suite of tests in parallel, i.e. (`focus=[Conformance]|network` with ginkgo parallel 25), not just the network policy tests. 
Main problems and flakes comes when things interact.
Should I clarify it here or did we understand the same?

**Comment by aojea** [comment]
/hold
We had a similar precedent for ingress, adding a new framework for testing, and the decision was to add the new framework under a new repo in kubernetes-sigs, https://github.com/kubernetes-sigs/ingress-controller-conformance

You can read the thread here https://groups.google.com/g/kubernetes-sig-network/c/dAt78O-JUCo/m/vvOXt3-XCgAJ

Due to the characteristics of netpol, should not be this a better alternative?

**Comment by aojea** [comment]
/hold cancel
seems sig-testing is ok since is not really a new framework

**Comment by jayunit100** [comment]
Yes from yesterdays sig testing meeting the decision was to just add as is, not a new global framework so it's ok .

File: keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md [hunk 2]
```diff
@@ -497,6 +497,13 @@ An architectural change to the current testing policies has been implemented and
 
 *This implementation runs continuously as part of the Antrea CNI project.*
 
+In order to avoid disruptions and minimize risks and/or regressions, new and old implementation will live in parallel during some time:
+
+- The new network policy framework will be added in a new folder `kubernetes/test/e2e/network/netpol`
+- The new network policy framework will use and additional tag [Feature:Netpol]
```
[the remark this answers is not in the document]
Comment (aojea) [reply keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md:503]: yeah, it was my fault, an excess of precaution, just wanted to have a mechanism for filtering, but the test have `Netpol` in the title so people can use it in a regex

[the remark this answers is not in the document]
Comment (aojea) [reply keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md]: done PTAL

[the remark this answers is not in the document]
Comment (aojea) [reply keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md]: @jayunit100 PTAL too, I added the change to use the same tags, you are using Netpol in the title so there is no reason to use a different tag, my apologies

[the remark this answers is not in the document]
Comment (jayunit100) [reply keps/sig-network/20200204-NetworkPolicy-verification-rearchitecture.md]: Sure that works thx :)

**Pull request closed without merging**